### DISCUSSION TITLE
Group discussion 25/03/66

### DISCUSSION THREAD

**Discussion by jaturaorn** [genarationth/sprintzero-d#6] [General]
เปลี่ยนจาก tracking เป็น todo-list ถ้ามีเปลี่ยนแปลงเพิ่ม ให้ค้นหาข้อมูลเพิ่ม, ถ้า todo-list เกินให้ซ่อน , menu เป็น eng คุณกิ๊ฟหาเพิ่มเติม,หน้า workout recipesตามงานคุณแป้ง(ตามแล้ว), page edit profile same like github, act ค้นหาข้อมูลเพิ่มเติม,content ในlanding page do like in figma

**Comment by sdsarun** [comment]
WOW!
